frontend/src/lib/components/PropertyFilters/PropertyFiltersDisplay.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { PropertyFilter } from '~/types'
+import PropertyFilterButton from './PropertyFilterButton'
+
+type Props = {
+    filters: PropertyFilter[]
+}
+
+const PropertyFiltersDisplay: React.FunctionComponent<Props> = ({ filters }: Props) => {
```

Comment:
I just saw your comment on the original ticket and I agree, it might be much nicer to allow people to change these filters directly in the table. It'll also get rid of this extra component. I'll leave it up to you whether you want to do this or not :)

Reply:
After I just implemented the read-only filter, it seems better. I'll remain it read-only :)

Reply:
Works for me!